app/serializers/watched_word_list_serializer.rb

```diff
@@ -4,7 +4,9 @@ class WatchedWordListSerializer < ApplicationSerializer
   attributes :actions, :words, :regular_expressions, :compiled_regular_expressions
 
   def actions
-    WatchedWord.actions.keys
+    WatchedWord.actions.keys.filter do |key|
+      SiteSetting.tagging_enabled || key != :tag
+    end
```

Comment:
How about we only filter the actions when tagging is disabled?

```suggestion
   SiteSetting.tagging_enabled ? WatchedWord.actions.keys : WatchedWord.actions.keys.filter { |k| k != :tag }
```